packages/sinuous/h/src/index.d.ts

```diff
@@ -11,31 +11,34 @@ declare namespace _h {
         Record<string, unknown>
       | null,
     ...children: ElementChildren[]
-  ): HTMLElement;
+  ): HTMLElement | SVGElement;
   function h(
     type: FunctionComponent,
     props:
       | (JSXInternal.HTMLAttributes | JSXInternal.SVGAttributes) &
         Record<string, unknown>
       | null,
     ...children: ElementChildren[]
-  ): HTMLElement;
+  ): HTMLElement | SVGElement;
   function h(
-    children: ElementChildren[]
+    ...children: ElementChildren[]
   ): DocumentFragment;
   namespace h {
     export import JSX = JSXInternal;
   }
 }
 
+type Frag = { _startMark: Text }
+type Value = Node | DocumentFragment | string | number
+
 export interface HyperscriptApi {
   // Hyperscript
   h: typeof _h.h;
 
   // Internal API
-  insert<T>(el: Node, value: T, endMark?: Node, current?: T, startNode?: Node): T;
+  insert<T>(el: Node, value: T, endMark?: Node, current?: T | Frag, startNode?: Node): T;
   property(el: Node, value: unknown, name: string, isAttr?: boolean, isCss?: boolean): void;
-  add(parent: Node, value: Node | string, endMark?: Node): Node;
+  add(parent: Node, value: Value | Value[], endMark?: Node): Node | Frag;
```

Comment:
So... `Frag` is mostly an internal concept, but for anyone writing wrappers (aka anyone actually bothering to touch `api.add`) it's probably good to mention